test/model.test.js

```diff
@@ -0,0 +1,893 @@
+    var model1 = thinky.createModel(util.s8(), {
+      id: String, name: String
+    });
+
+    var model2 = thinky.createModel(util.s8(), {
+      id: String, name: String
+    });
+
+    return Promise.all([ model1.ready(), model2.ready() ])
+      .then(function() {
+        expect(model1).to.not.eql(model2);
+      });
+  });
+
+  it('should throw error for non existent table access', function() {
+    var model = thinky.createModel("nonExistentTable", {
+      id: String, name: String
+    }, {
+      init: false
+    });
+
+    return expect(model.get(1).run())
+      .to.eventually.be.rejectedWith(/^Table `.*` does not exist in/);
+  });
+});
+
+describe('[_]getModel', function() {
+  it('_getModel', function() {
+    var name = util.s8();
+    var model = thinky.createModel(util.s8(), {
+      id: String, name: String
+    }, {
+      init: false
+    });
+
+    expect(model._getModel()).to.have.property('_name');
+  });
+
+  it('getTableName', function() {
+    var modelName = util.s8();
+    var model = thinky.createModel(modelName, {
+      id: String, name: String
+    }, {
+      init: false
+    });
+
+    expect(model).to.have.property('getTableName');
+    expect(model.getTableName()).to.equal(modelName);
+  });
+});
+
+describe('Model', function() {
+  before(function() {
+    test.models.Model = thinky.createModel(util.s8(), { str: String });
+    return test.models.Model.ready();
+  });
+  after(function() { return test.dropTables(); });
+  afterEach(function() { return test.clearTables(); });
+
+  it('should create a new instance of a model', function() {
+    var str = util.s8();
+    var doc = new test.models.Model({ str: str });
+
+    expect(util.isPlainObject(doc)).to.be.true;
+    expect(doc.str).to.eql(str);
+  });
+
+  it('should create multiple instances from the same document', function() {
+    var str = util.s8();
+    var num = util.random();
+
+    var values = { str: str, num: num };
+    var doc = new test.models.Model(values);
+    var otherDoc = new test.models.Model(values);
+
+    expect(doc).to.eql(values);
+    expect(doc).to.not.eql(otherDoc);
+
+    doc.str = doc.str + util.s8();
+    expect(doc.str).to.not.eql(otherDoc.str);
+
+    var anotherDoc = new test.models.Model(values);
+    expect(anotherDoc).to.not.eql(otherDoc);
+    expect(anotherDoc).to.not.eql(doc);
+  });
+
+  it('should create two distinct instances with the same options', function() {
+    var options = { str: util.s8() };
+    var doc1 = new test.models.Model(options);
+    var doc2 = new test.models.Model(options);
+
+    expect(doc1).to.not.eql(doc2);
+  });
+
+  it('should ensure two instances are different', function() {
+    var str1 = util.s8(), str2 = util.s8();
+    var doc1 = new test.models.Model({ str: str1 });
+    expect(doc1.str).to.equal(str1);
+
+    var doc2 = new test.models.Model({ str: str2 });
+    expect(doc2.str).to.equal(str2);
+    expect(doc1.str).to.equal(str1);
+    expect(doc1).to.not.eql(doc2);
```

Comment:
I think you've made your point with these, they can all be replaced with 'equal' (http://chaijs.com/api/bdd/ `equal` section indicates a strict === equality). Problem solved